vue-app/src/components/BrightIdWidget.vue

```diff
@@ -43,21 +42,20 @@
   </div>
   <div v-else class="setup-container">
     <div class="row">
-      <div v-if="isVerified">
+      <div v-if="isLinked && isRegistered && isSponsored && isVerified">
         <icon-status
           v-bind:happy="true"
           logo="brightid.svg"
           secondaryLogo="checkmark.svg"
         />
       </div>
-      <div v-if="!isVerified">
+      <div v-else>
         <icon-status
           v-bind:sad="true"
           logo="brightid.svg"
           secondaryLogo="close-black.svg"
         />
       </div>
-      <!-- TODO: should this icon demo verified status or completion of entire setup? Is it even needed?-->
```

Comment:
(I know you didn't implement this) this class logic below is a bit sloppy:
```
      <p>
        <span
          :class="{
            step0: !isLinked && !isVerified && !isSponsored && !isRegistered,
            step1: isLinked,
            step2: isVerified || isSponsored,
            step3: isVerified && isSponsored,
            step4: isRegistered,
          }"
        />
        / 4
      </p>
```
We don't need CSS here, we should just calculate the number step we're at. @pettinarip implemented this basic logic here: https://github.com/ethereum/clrfund/blob/brightid-integration-2/vue-app/src/views/Verify.vue#L326

Then we can just display {{currentStep +1}} in the UI.

Reply:
Sounds good 👍